packages/video_player/pubspec.yaml

```diff
@@ -2,8 +2,8 @@ name: video_player
 description: Flutter plugin for displaying inline video with other Flutter
   widgets on Android and iOS.
 author: Flutter Team <flutter-dev@googlegroups.com>
+version: 0.5.5
```

Comment:
I think we should make this a 0.6.0 to allow for patches to the version of the plugin that still uses `MediaPlayer`.